.github/workflows/frontend-ci.yaml

```diff
@@ -0,0 +1,33 @@
+# Setup an action to run the frontend CI for PRs
+
+# name of the action
+name: Frontend CI
+
+# when should it run
+on:
+  pull_request:
+    branches:
+      - main
+
+# what should it do
+jobs:
+  # name of the job
+  build:
+    # what should it run on
+    runs-on: ubuntu-latest
+    # default values for all the steps
+    defaults:
+      run:
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+      - name: Install dependencies
+        run: npm install
```

Comment:
```suggestion
        run: npm ci
```

Reply:
We use this most of the times. https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci

Reply:
@Hsbalazs Can you complete this change requested by Mark?